docs/fallbacks.md

```diff
@@ -1,28 +1,66 @@
 # Fallbacks
 
-### Circuit Breaker
+## Circuit Breaker
 
 A simple implementation of [Circuit Breaker](http://martinfowler.com/bliki/CircuitBreaker.html) pattern.
 
+### MaxFailuresRule
```

Comment:
What about moving the rule descriptions to the bottom and put it under a h2 title `Rule`? I think it would be better for the reader to understand how the `CircuitBreaker` class works and read about the `Rule` obtions later.